app/src/androidTest/java/com/github/polypoly/app/commons/PolyPolyTest.kt

```diff
@@ -69,27 +73,27 @@ abstract class PolyPolyTest(
 
         val TEST_GAME_LOBBY_FULL = GameLobby(
             TEST_USER_0, GameParameters(GameMode.RICHEST_PLAYER, 2, 6,
-            60, 20, emptyList(), 100), "Full gameLobby", "lobby1234"
+            60, 20, getZones(), 100), "Full gameLobby", "lobby1234"
```

Comment:
You do not need to specify the `gameMap` parameter when it contains all the zones because it is by default initialized to `getZone()`

Reply:
Indeed I didn't notice that, however if I only exclude this parameter it gives
`kotlin
GameParameters(GameMode.RICHEST_PLAYER, 4, 6, 360, 20,  initialPlayerBalance =  300)
`
which I find quite ugly.
Moreover, `GameMode.RICHEST_PLAYER` is actually also the default value but it is kept here as it is more meaningful.
Finally, as it is an instance specific for tests I prefer a being a bit more verbose to explicit its intent.

Reply:
Since we use "magic numbers", specifying all the parameters is not a bad idea either